conf/steps/validation.config

```
@@ -65,6 +66,7 @@ process {
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
+        tag = {"${meta.id} ${meta.panel} ${meta.tools}"}
```

```suggestion
        tag = {"${meta.id} ${meta.panel}"}
```
The meta map here shouldn't have the tools yet.